indico/modules/events/management/client/js/index.js

```diff
@@ -17,6 +17,10 @@
 
 /* global setupSearchBox:false, strnatcmp:false */
 
+import 'selectize';
```

Comment:
If we import this in management now, can we remove it from vc? The VC pages using it are all in management so I think we are just duplicating the CSS there.